cli/lib/kontena/scripts/completer

```diff
@@ -3,238 +3,7 @@
 
 # resolve bin path, ignoring symlinks
 require 'pathname'
-require 'yaml'
 bin_file = Pathname.new(__FILE__).realpath
+$:.unshift File.expand_path('../../..', bin_file)
```

Comment:
Looking at the context, this is replacing `$:.unshift File.expand_path('../../lib', bin_file)` -> `$:.unshift File.expand_path('../../..', bin_file)`, that supposed to be different?